ui/admin/app/controllers/scopes/scope/targets/index.js

```diff
@@ -91,4 +97,225 @@ export default class ScopesScopeTargetsIndexController extends Controller {
     this[paramKey] = [...selectedItems];
     this.page = 1;
   }
+
+  // =actions
```

Comment:
`// =actions` is also on line 75. Should we remove this one?

Reply:
yes, great catch!